docs/ref/models/options.txt

```diff
@@ -288,6 +288,10 @@ Django quotes column and table names behind the scenes.
     incurs a cost to your database. Each foreign key you add will
     implicitly include all of its default orderings as well.
 
+    If a query doesn't have an ordering specified, results are returned from
+    the database in an unspecified order. A particular ordering is only
+    guaranteed if some ordering is specified.
```

Comment:
What I mean is that at least one element of the set of the unique constraint defined on the table **must** be included in the set of fields used for ordering.

For example in the case of:

```python
class Foo(models.Model):
    bar = models.CharField(max_length=100, unique=True)
    baz = models.CharField(max_length=100, unique=True)
    bat = models.CharField(max_length=100, unique=True)

    class Meta:
        unique_together = {('baz', 'bat')}
```

The ordering must include either `pk`, `id`, `bar` or `{'baz', 'bat'}` a deterministic ordering. I remembered a related ticket in the admin, see d636150e533a2954f9b74aa6ca2e7375b18437ef.